### ISSUE TITLE
Could not use `Explain code step by step` with text selected in Neovim

### ISSUE THREAD

**Issue by vitorf7** [MassimilianoPasquini97/raycast_ollama#7]
Thank you for the extension. 
Recently while programming in Neovim I selected a piece of code using the `v-line` to select all the code I wanted to ask my `llama2` to explain in order to test the extension.

However, when I selected it and tried to use the explain the code step by step it did not recognise any text as being selected.

Code I had selected:
![ollama-neovim-issue](https://github.com/MassimilianoPasquini97/raycast_ollama/assets/469102/860ca316-a8cb-4909-999e-6044c034b865)

Results from the extension:
![Screenshot 2023-11-08 at 12 07 01](https://github.com/MassimilianoPasquini97/raycast_ollama/assets/469102/d02eb4a4-a392-45d8-98c2-2ad27d571f35)

However, if I copied it to say `TextEdit` and then selected it with CMD + a and asked the extension it worked.

```
MacOS Sonoma 14.1
Wezterm 20230712-072601-f4abf8fd
NVIM v0.10.0-dev-1413+g684e93054-Homebrew
```

When I tried it running Neovim in `Alacritty 0.12.3` the extension would say that no text is selected.
![Screenshot 2023-11-08 at 12 21 08](https://github.com/MassimilianoPasquini97/raycast_ollama/assets/469102/e68ccd5b-fe25-4cf0-948e-378861e90d16)

**Comment by vitorf7** [comment]
This is not limited to the `explain code` as I tried to fix spelling, etc and the same happens. Something in terms of the communication between Raycast and Neovim is not picking up the text which seems to behave differently based on different terminals since Wezterm at I got something back from the extension (albiet saying nothing was found) whereas with alacritty it says no text was selected

**Comment by MassimilianoPasquini97** [comment]
Hi @vitorf7,
nvim selection is not passed to the os so is not readable from raycast extension. This issue was solved on #4 with the possibility to enable fallback to the clipboard or use clipboar as a primary input source instead of text selection.

![image](https://github.com/MassimilianoPasquini97/raycast_ollama/assets/62497822/63be9aba-8e2f-432d-a3c3-2efb7423d272)

Remember to edit nvim config file for enable copied text to be passed to the os clipboard.

```nvim
set clipboard=unnamedplus   	" using system clipboard
```
